webcrypto/subtle_crypto.go

```diff
@@ -263,37 +269,14 @@ func (sc *SubtleCrypto) Sign(algorithm, key, data goja.Value) *goja.Promise {
 			return
 		}
 
-		// 10.
-		switch normalized.Name {
-		case HMAC:
-			keyAlgorithm, ok := ck.Algorithm.(hasHash)
-			if !ok {
-				reject(NewError(InvalidAccessError, "key algorithm does not describe a HMAC key"))
-				return
-			}
-
-			keyHandle, ok := ck.handle.([]byte)
-			if !ok {
-				reject(NewError(InvalidAccessError, "key handle is of incorrect type"))
-				return
-			}
-
-			hashFn, ok := getHashFn(keyAlgorithm.hash())
-			if !ok {
-				reject(NewError(NotSupportedError, "unsupported hash algorithm "+keyAlgorithm.hash()))
-				return
-			}
-
-			hasher := hmac.New(hashFn, keyHandle)
-			hasher.Write(dataToSign)
-
-			// 10.
-			mac := hasher.Sum(nil)
-
-			resolve(rt.NewArrayBuffer(mac))
-		default:
-			reject(NewError(NotSupportedError, fmt.Sprintf("unsupported algorithm %q", normalized.Name)))
+		signature, err := signer.Sign(ck, dataToSign)
+		if err != nil {
+			log.Println(err.Error())
```

Comment:
```suggestion
```

leftover debugging log?

Reply:
Good catch! we don't use a log, right? Maybe we could add this to the linter check?